infrastructure/iam_lambda_s3access.tf

```diff
@@ -0,0 +1,53 @@
+resource "aws_iam_policy" "nfish-des-pol-lambda_s3_access" {
+  name        = "nfish-des-pol-lambda_s3_access"
+  description = "access to s3 for lambda"
+  path        = "/"
+
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:*",
+                "s3-object-lambda:*"
+            ],
+            "Resource": "*"
+        }
```

Comment:
I would give only access to the resources it needs. 
Try to specify the Resource and the action

Reply:
ok